woodpecker-agent/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: woodpecker-agent
 description: A Helm chart for the Woodpecker agent
 type: application
-version: <version>
-appVersion: "v<version>"
+version: 1.0.0
+appVersion: "next"
```

Comment:
This way a stable release would use `next` by default. Instead I would simply release a proper new version to the charts registry and let users set the appVersion if they want to use `next` via the values.

Reply:
Fair enough! Yet I'd say that `appVersion` does not have much of an influence right now because if someone wants "next", the image tags must be adapted anyhow. 

So right now I guess the chart should set

- 0.15.6 as both `version` and `appVersion`? Or use it's own versioning for the chart? 
- The README should state the need to adapt `image.tag` to actually use "next" (on the application side) unless there is a logic for mapping `appVersion` to `image.tag`?

If someone installs the chart from git instead of the latest chart release, having `0.15.6` there as the version is probably misleading. So maybe something like `1.0.0-rc`? Or `99.999` to indicate dev?

Whatever you prefer, I think the actions needed to install stable/next should be clearly outlined in the README as it becomes confusing otherwise. And combining the latest chart-release with the `"next"` tags of the app is not really working.

Reply:
The roadmap I am currently having in mind:
Combine both charts into one #5 as most users will have a single kubernetes cluster with a single agent that can scale. Add an option to disable the server / agent part to allow users to just deploy another agent if needed. This new chart allow us to start a new versioning without a breaking change. So we can start with version 0.0.1 and update the chart independently using that version. The appVersion I would set to the latest stable `0.15.6` and allow the user to adjust it if wanted.

Reply:
Sounds like a plan. Yet it also sounds that it might take some time. Hence I wonder what to do in the meantime/now as the current placeholders don't work with `helm-git` and other tools. 

I'm not bound to this PR but I think the placeholders should be replaced somewhat soonish, what do you think?